packages/pwa/app/pages/cart/partials/cart-secondary-button-group.jsx

```diff
@@ -23,12 +23,22 @@ import {noop} from '../../../utils/utils'
 import useCustomer from '../../../commerce-api/hooks/useCustomer'
 
 export const REMOVE_CART_ITEM_CONFIRMATION_DIALOG_CONFIG = {
-    dialogTitle: defineMessage({defaultMessage: 'Remove Item?'}),
+    dialogTitle: defineMessage({
+        defaultMessage: 'Remove Item?',
+        id: 'global.cart.title.remove_item'
```

Comment:
Left a comments for the ids [above](https://github.com/SalesforceCommerceCloud/pwa-kit/pull/239/files#r758763369)

Reply:
Yeah, let's replace these 'global' ids because they're not as global as I initially thought 😅